src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,44 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(health) {
+    const FULL_HEALTH = 100;
+
+    this.health = health;
+
+    this.health = FULL_HEALTH;
+    Animal.alive.push(this);
+  }
+  static animals() {
+    Animal.alive = Animal.alive.filter((animal) => animal.health > 0);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super();
+    this.hidden = false;
+  }
+  hide() {
+    this.hidden = true;
+  }
 }
-
 class Carnivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super();
+  }
+  bite(target) {
+    const BITE_CARNIVORE = 50;
```

Comment:
`BITE_CARNIVORE` is not the best naming for this constant. I'd suggest to name it something like `HEALTH_REDUCTION_ON_BITE`, so that it clearly describes the meaning of constant.